app/src/main/java/com/android/unio/ui/explore/Explore.kt

```diff
@@ -227,17 +229,22 @@ fun ExploreScreenContent(
 fun AssociationItem(association: Association, onClick: () -> Unit) {
   Column(
       modifier =
-          Modifier.clickable(onClick = onClick).testTag("associationItem_${association.name}")) {
+          Modifier.clickable(onClick = onClick)
+              .testTag(ExploreContentTestTags.ASSOCIATION_ITEM + association.name)) {
         /**
-         * AdEC image is used as the placeholder. Will need to add the actual image later, when the
-         * actual view model is used.
+         * AdEC image is used as the placeholder. Will need to remove it when all images are
+         * available on the Firestore database
```

Comment:
I don't think we should remove the placeholder image, rather change it to something more appropriate than the AdEC image. As we should have a placeholder image in case an association doesn't have an image available,

Reply:
You're right. We should define one asap for MS2.